bootstrap.py

```diff
@@ -375,21 +402,48 @@ def return_matching_org_label(organization):
     return org_label
 
 
+def return_matching_host(fqdn):
+    # Given an org, find its id.
+    myurl = "https://" + options.sat6_fqdn + ":" + API_PORT + "/api/v2/hosts/?" + urlencode([('search', 'name=%s' % fqdn)])
+    if options.verbose:
+        print myurl
+    hosts = get_json(myurl)
+    if options.verbose:
+        print json.dumps(hosts, sort_keys = False, indent = 2)
+    if len(hosts['results']) == 1:
+        host_id = hosts['results'][0]['id']
+        return host_id
+    elif len(hosts['results']) == 0:
+        return None
+    else:
+        print_error("Found too many hosts with same name %s" % fqdn)
+        sys.exit(2)
+
+
 def create_host():
     myhgid = return_matching_hg_id(options.hostgroup)
     mylocid = return_matching_location(options.location)
     myorgid = return_matching_org(options.org)
     mydomainid = return_matching_domain_id(DOMAIN)
     architecture_id = return_matching_architecture_id(ARCHITECTURE)
-    if options.verbose:
-        print "------\nmyhgid: " + str(myhgid) + "\nmylocid: " + str(mylocid) + "\nmyorgid: " + str(myorgid) + "\nMAC: " + str(MAC) + "\nDOMAIN_ID: " + str(mydomainid) + "\nARCHITECTURE_ID: " + str(architecture_id) + "\n------"
+    host_id = return_matching_host(FQDN)
```

Comment:
Sorry, i have not got it.
In my last revision, there is the guessing on the ID before 
`    if options.force and host_id is not None:`
So the host_id have to be guessed in first place.

Better would be changing the code heavily, making in first place the check about managed/unmanaged.
If managed, architecture, operatingsystem, ... have to be retrieved.
As well, only if force = true, we sould make the other steps.
Also, i think that we should use only one function to get "ID", because the function are very similar.

But i would like to do it in the next pull request :-)